src/homework/homework02/Student.java

```diff
@@ -1,24 +1,41 @@
 package homework.homework02;
 
 public class Student extends Person{
-    private int semester;
+    protected Faculty faculty;
+    protected int semester;
+    protected Person person;
```

Comment:
No to w końcu jak to jest:
* Z jednej strony klasa Student rozszerza klasę Person. Czyli wśród swoich atrybutów posiada atrybuty Person - pesel, name itp. Można napisac: `super.getPesel()` i dostanie się pesel samego siebie (a że Person.pesel nie nie prywatny, to można nawet zrobić `this.pesel = 1500`)
* Z drugiej strony klasa Student posiada atrybut (składową) Person, i to jest całkiem inny Person niż ten odziedziczony.
```Java
super.setPesel(1500);
this.setPesel(100);
assertEquals(1500,pesel);
assertEquals(100,person.pesel);
```